src/devices/multipleanalogsensorsserver/MultipleAnalogSensorsServer.h

```diff
@@ -49,6 +50,10 @@ class MultipleAnalogSensorsServer :
     // Generic vector buffer
     yarp::sig::Vector m_buffer;
 
+    // Wrapped subdevices, if any
+    yarp::dev::PolyDriver subdevice;
```

Comment:
It would be nice to keep the `m_` prefix for consistency with the rest of the attributes, thanks!